main.go

```diff
@@ -33,8 +32,8 @@ func info() string {
 
 // main function
 func main() {
-	var keytab string
-	flag.StringVar(&keytab, "keytab", "", "keytab file to check")
+	var keytabF string
```

Comment:
I'm not sure why did you changed variable name, but please do not use short letters, it should be either `keytab` or `keytabFile`

Reply:
It was conflicting with `gopkg.in/jcmturner/gokrb5.v8/keytab` because of same names. I changed it to `keytabFile`.